src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Flex, Text, StackDivider, HStack, Link, Image } from '@chakra-ui/react';
+import facebook from '../../assets/facebook.png';
+import instagram from '../../assets/instagram.svg';
+import twitter from '../../assets/twitter.png';
+
+const Footer = () => {
+  return (
+    <Flex
+      as="footer"
```

Comment:
maybe needs position: absolute and bottom: 0 so it is at the bottom of the page?
[reference](https://www.freecodecamp.org/news/how-to-keep-your-footer-where-it-belongs-59c6aa05c59c/)